schemas/config-editor.xsd

```diff
@@ -1007,6 +1007,7 @@ the mandatory section with no name and then the inner elements.
       <xs:attribute ref="or"/>
       <xs:attribute ref="in"/>
       <xs:attribute ref="displayIfRecord"/>
+      <xs:attribute ref="displayIfServiceInfo"/>
```

Comment:
I would add some documentation to explain a bit this attribute, with the name only I think can be a bit confusing if refers to some service metadata property.

Reply:
https://github.com/geonetwork/core-geonetwork/blob/main/schemas/config-editor.xsd#L1650-L1665 is not enough ?

Reply:
Yes, that's enough. Apologies, I missed that, seem it was added in a previous commit.